Makefile

```diff
@@ -10,35 +10,33 @@ MJKEY_PATH ?= ~/.mujoco/mjkey.txt
 # is already created
 CONFIG_PERSONAL := garage/config_personal.py
 
-build-ci: TAG ?= rlworkgroup/garage-ci
-build-ci: docker/docker-compose-headless.yml copy_config_personal
+build-ci: TAG ?= rlworkgroup/garage-ci:latest
```

Comment:
any reason you called these latest? those are usually just tags for docker hub.

Reply:
It was just the default in case no TAG is provided. Docker will add latest as version anyway if the TAG is  `rlworkgroup/garage-ci`, right?